tests/Parser/DocumentParserTest.php

```diff
@@ -42,7 +45,11 @@ public function testErrorWhenDirectiveThrowsException(): void
             ->method('getName')
             ->willReturn('code-block-name');
 
-        $environment->expects(self::once())
+        $environment->expects(self::any())
```

Comment:
I think `->expects(self::any())` achieves nothing. Let's drop it and call `method()` directly?

```suggestion
        $environment
```